browser/payments/payments_service_impl.cc

```diff
@@ -238,6 +238,10 @@ void PaymentsServiceImpl::OnWalletCreated(ledger::Result result) {
   TriggerOnWalletCreated(result);
 }
 
+void PaymentsServiceImpl::OnWalletProperties(ledger::WalletProperties result) {
```

Comment:
we don't want to expose `ledger` outside of the payments service. What exactly is in wallet properties?

Reply:
this are the fields https://github.com/brave-intl/bat-native-ledger/pull/26/files#diff-2779bd349aa484b3cafcf4c1a138efd5R22. We use them all in UI, like balance, grants, range and choices for monthly contribution

Reply:
I think we have individual methods for the balance and we should add them for anything else you need

Reply:
but we should never directly expose anything in `ledger`. For Grants you might want to do something like `ContentSite` to expose whatever fields we need in the UI

Reply:
basically we should only expose the things in payment service that we actually use in the brave rewards ui and leave all the internal details inside of ledger

Reply:
if any kind of translation needs to be made to put it in the format we want for the user, that translation should happen in the payment service